Update native development template to Python 3.7

`PythonVersion` property should be changed to 3.7, since that's the version we include with dev16.

https://github.com/Microsoft/PTVS/blob/master/Python/Templates/NativeDevelopment/ProjectTemplates/VC/Python/PythonExtensionModule/PythonTemplate.vcxproj

Maybe the Windows SDK version needs an update too.
```
1>C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.Cpp.WindowsSDK.targets(46,5): error MSB8036: The Windows SDK version 10.0.16299.0 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution".
```

--- a/Python/Templates/NativeDevelopment/ProjectTemplates/VC/Python/PythonExtensionModule/PythonTemplate.vcxproj
+++ b/Python/Templates/NativeDevelopment/ProjectTemplates/VC/Python/PythonExtensionModule/PythonTemplate.vcxproj
@@ -21,8 +21,8 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>$guid1$</ProjectGuid>
     <RootNamespace>$safeprojectname$</RootNamespace>
-    <PythonVersion>3.6</PythonVersion>
-    <WindowsTargetPlatformVersion>10.0.16299.0</WindowsTargetPlatformVersion>
+    <PythonVersion>3.7</PythonVersion>
+    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">

